src/interactive/plots.ts

```diff
@@ -42,7 +42,7 @@ export function showPlotPane() {
     const plotTitle = g_plots.length > 0 ? `Julia Plots (${g_currentPlotIndex + 1}/${g_plots.length})` : 'Julia Plots (0/0)'
     if (!g_plotPanel) {
         // Otherwise, create a new panel
-        g_plotPanel = vscode.window.createWebviewPanel('jlplotpane', plotTitle, { preserveFocus: true, viewColumn: vscode.ViewColumn.Active }, { enableScripts: true })
+        g_plotPanel = vscode.window.createWebviewPanel('jlplotpane', plotTitle, { preserveFocus: true, viewColumn: vscode.ViewColumn.Beside }, { enableScripts: true })
```

Comment:
```suggestion
    g_plotPanel = vscode.window.createWebviewPanel(
        'jlplotpane',
        plotTitle,
        {
            preserveFocus: true,
            viewColumn: vscode.ViewColumn.Beside
        },
        { 
            enableScripts: true
        }
    )
```

Just cosmetic change.